data/data-pipeline/data_pipeline/score/score_a.py

```diff
@@ -0,0 +1,13 @@
+from data_pipeline.score.score import Score, FN, pd, logger
+
+
+class ScoreA(Score):
+    def add_columns(self) -> pd.DataFrame:
+        logger.info("Adding Score A")
+        self.df["Score A"] = self.df[
```

Comment:
Nit: should we move `"Score A"` (and the names of all the other scores) into a field name in the constants?

Reply:
I thought about that! I opted out of it as out of scope but I'm happy to move them in.